### PR TITLE
update setup.py to latest pip

### PR DESCRIPTION
**Pull request by arnaudon**
Getting rid of pip.req, as done in Brayns

### CODE REVIEW

**Comment by wizmer** [comment]
You may as well want to get rid of requirements.txt and put then in the setup.py

File: python/setup.py [hunk 1]
```diff
@@ -22,38 +22,37 @@
...
 
 
 REQS = parse_reqs(os.path.join(BASEDIR, "requirements.txt"))
 
+
 # read the contents of README.md
 this_directory = os.path.abspath(os.path.dirname(__file__))
-with open(os.path.join(this_directory, 'README.md')) as f:
+with open(os.path.join(this_directory, "README.md")) as f:
     long_description = f.read()
```
Comment (wizmer) [inline python/setup.py:51]: I don't think you need to append "this_directory"

```python
with open('README.md') as f:
    long_description = f.read()
```
Seems to be enough in MorphIO.

File: python/setup.py [hunk 2]
```diff
@@ -22,38 +22,37 @@
...
-except ImportError: # for pip <= 9.0.3
-    from pip.req import parse_requirements
-    from pip.download import PipSession
-from optparse import Option
+import pathlib
+import pkg_resources
+import setuptools
+from setuptools import find_packages, setup
 
 BASEDIR = os.path.dirname(os.path.abspath(__file__))
```
Comment (wizmer) [inline python/setup.py:30]: We will no longer need this once you drop requirements.txt

**Comment by ppodhajski** [comment]
fixed in https://github.com/BlueBrain/Rockets/pull/88

**Pull request closed without merging**